haskell-src/Concordium/Types/Accounts.hs

```diff
@@ -569,6 +574,30 @@ instance HashableTo (AccountStakeHash 'AccountV4) (AccountStake 'AccountV4) wher
                             put _delegationPendingChange
                         )
 
+-- | The 'AccountV5' hashing of 'AccountStake' DOES NOT INCLUDE the staked amount.
+--  This is since the stake is accounted for separately in the @AccountHash@.
+instance HashableTo (AccountStakeHash 'AccountV5) (AccountStake 'AccountV5) where
+    getHash AccountStakeNone = accountStakeNoneHashV5
+    getHash (AccountStakeBaker AccountBaker{..}) =
+        AccountStakeHash $
+            Hash.hashLazy $
+                "A5Baker"
+                    <> runPutLazy
+                        ( do
+                            put _stakeEarnings
+                            put _accountBakerInfo
+                        )
+    getHash (AccountStakeDelegate AccountDelegationV1{..}) =
+        AccountStakeHash $
+            Hash.hashLazy $
+                "A5Delegation"
+                    <> runPutLazy
+                        ( do
+                            put _delegationIdentity
+                            put _delegationStakeEarnings
+                            put _delegationTarget
+                        )
+
```

Comment:
Can you please add some documentation about the pending change thing (probably in the `AccountV4` instance).